docs/_articles/en/v3.x/salesforce-graph-engine/try-it-yourself.md

```diff
@@ -4,131 +4,113 @@ lang: en
 ---
 
 
-## Before you get started
-SFGE works a bit differently from other analyzers with which you might be familiar. As such, we strongly encourage you
-to read about [the engine](./en/v3.x/salesforce-graph-engine/introduction) and [its capabilities](./en/v3.x/salesforce-graph-engine/working-with-sfge) before proceeding.
+## Prerequisites
 
-Also, so you can try these examples yourself, please install the pilot version of the analyzer as per [these instructions](./en/v3.x/getting-started/install).
+Salesforce Graph Engine (Graph Engine) works a bit differently from other analyzers. To get started with Graph Engine:
 
-Once you've done all that, continue reading here to see some basic SFGE operations, so you can get a sense of how it all works together.
+* Read the [Introduction to Graph Engine](https://forcedotcom.github.io/sfdx-scanner/en/v3.x/salesforce-graph-engine/introduction/).
+* Understand [Working with Graph Engine](https://forcedotcom.github.io/sfdx-scanner/en/v3.x/salesforce-graph-engine/working-with-sfge/).
+* Install [Salesforce Code Analyzer](https://forcedotcom.github.io/sfdx-scanner/en/v3.x/getting-started/install/).
 
-## See the rules
-First thing's first. Let's see the rules that exist in SFGE. Run the following command.
+Next, run a command to view Graph Engine rules, then install our sample project to try out Graph Engine for yourself.
+
+## See the Rules
+To see Graph Engine’s rules, run:
 
 ```bash
 sfdx scanner:rule:list --engine sfge
 ```
 
-Note that there's just the one rule. [ApexFlsViolationRule](./en/v3.x/salesforce-graph-engine/rules/#apexflsviolationrule)
-identifies CRUD/FLS vulnerabilities in your Apex code. In the future, more rules will likely be added, but for now there's
-just that rule. As such, all of the examples will focus on CRUD/FLS.
+Currently, just one rule is returned. [ApexFlsViolationRule](./en/v3.x/salesforce-graph-engine/rules/#apexflsviolationrule) identifies Create Read Update Delete/Field-Level Security (CRUD/FLS) vulnerabilities in your Apex code.
 
-## Let's look at the sample project
-All of our examples will be using the [sample app](https://github.com/forcedotcom/sfdx-scanner/tree/dev-3/test/code-fixtures/projects/sfge-working-app/force-app/main/default).
+## Clone the Sample Project
+All our examples use our [sample app](https://github.com/forcedotcom/sfdx-scanner/tree/dev-3/test/code-fixtures/projects/sfge-working-app/force-app/main/default).
 
-To begin:
+To begin, clone the `dev-3` branch of the repo.
```

Comment:
I can't remember whether we intend to merge `dev-3` into `dev`, but that may be somethign we do. If we do, this will need to change.